internal/graph/check.go

```diff
@@ -153,7 +154,7 @@ func (cc *ConcurrentChecker) checkInternal(ctx context.Context, req ValidatedChe
 	}
 
 	// Deduplicate any incoming resource IDs.
-	resourceIds := util.UniqueSlice(req.ResourceIds)
+	resourceIds := slicez.Unique(req.ResourceIds)
```

Comment:
`slicez`? Why not just `slices`?

Reply:
nah collision -- that's the idea with the z-suffix convention